src/wrapper/AirtopClient.ts

```diff
@@ -36,4 +43,12 @@ export class AirtopClient {
   error(err: any) {
     console.error(err);
   }
+
+	batchOperate = async (
```

Comment:
Personally I think that might be confusing since we'd potentially use a combination of multiple sessions/windows.  Right now you'd invoke the batch operator as follows.

```
await client.batchOperate(urls, myFn,  myConfig);
```

Happy to change that if others agree it belongs elsewhere though.